graphix/pattern.py

```diff
@@ -2003,7 +2002,7 @@ def pauli_nodes(pattern: Pattern, leave_input: bool):
     return pauli_node, non_pauli_node
 
 
-def is_pauli_measurement(cmd: command.Command, ignore_vop=True):
+def is_pauli_measurement(cmd: command.Command):
```

Comment:
Let's add return type annotation (would be `Literal[XXX] | None`)

Reply:
I changed `measure_pauli` to use `Pauli` directly instead of relying on a string encoding in 654f300: the return type is now just `Pauli`.